test/ConsoleLogger.test.js

```diff
@@ -26,6 +26,7 @@ const serviceResults = [
 ];
 
 test.before(t => sinon.stub(console, 'log'));
+test.before(t => sinon.stub(process.stdout, 'write'));
```

Comment:
need to also restore `process.stdout.write.restore()` so we don't affect other tests